lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+from flatten_json import flatten
+
+
+def lusid_respose_to_pd(lusid_response):
+
+    """
+    This function takes a LUSID API response and attempts to convert the response into a Pandas DataFrame or Series.
+    Not all LUSID API responses are the same datatype.
+    Therefore we need to implement some if/else conditional logic to see how the response should be converted.
```

Comment:
This comment is more an implementation detail rather than user documentation, I would move to inside the function

Reply:
Thanks. Corrected in latest commit.